code/form/LoginForm.php

```diff
@@ -84,8 +94,9 @@ public function Actions()
         $actions = parent::Actions();
         $fields = $this->Fields();
 
-        // Remove the lost-password action from the TOTP token form
+        // Remove the lost-password action from the TOTP token form and insert a cancel button
```

Comment:
Seems like a good idea... common sense that once you've hit TOTP you've probably already remembered your password 👍